File: framework/ui-elements/src/displays/components/DocumentEditor.js
import React from "react";
import AbstractDocumentEditor from "../../../gen/displays/components/AbstractDocumentEditor";
import DocumentEditorNotifier from "../../../gen/displays/notifiers/DocumentEditorNotifier";
import DocumentEditorRequester from "../../../gen/displays/requesters/DocumentEditorRequester";

const styles = theme => ({});

export default class DocumentEditor extends AbstractDocumentEditor {

	constructor(props) {
		super(props);
		this.notifier = new DocumentEditorNotifier(this);
		this.requester = new DocumentEditorRequester(this);
	};

    render() {
        return (<div>{this.translate("Indicate editor type by using facets in Konos file")}</div>);
    };

}

File: framework/ui/src/io/intino/alexandria/ui/displays/components/DocumentEditor.java
package io.intino.alexandria.ui.displays.components;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.DocumentEditorNotifier;

public abstract class DocumentEditor<DN extends DocumentEditorNotifier, B extends Box> extends AbstractDocumentEditor<DN, B> {

	public DocumentEditor(B box) {
		super(box);
	}

}